nodeup/pkg/model/docker.go

```diff
@@ -556,8 +556,8 @@ var dockerVersions = []dockerVersion{
 		Hash:          "b4ce72e80ff02926de943082821bbbe73958f87a",
 		ExtraPackages: map[string]packageInfo{
 			"container-selinux": {
-				Version: "2.68",
-				Source:  "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm",
+				Version: "2.107",
+				Source:  "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-3.el7.noarch.rpm",
 				Hash:    "d9f87f7f4f2e8e611f556d873a17b8c0c580fec0",
```

Comment:
Update the sha1 value to "a837021203770bdf025ffc0584940575a3195e36" in all four places.